templates/core/core-pre-upgrade-job.yaml

```diff
@@ -47,6 +47,9 @@ spec:
               secretKeyRef:
                 name: {{ .Values.database.external.existingSecret }}
                 key: password
+        {{- end }}
+        {{- if not (empty .Values.core.containerSecurityContext) }}
+        securityContext: {{ .Values.core.containerSecurityContext | toYaml | nindent 10 }}
```

Comment:
Shouldnt this use `.Values.containerSecurityContext` instead of `.Values.core.containerSecurityContext`?
`.Values.core.containerSecurityContext` does not exists in `values.yaml`

Reply:
good catch :+1: